tests/test_cmd_grid.sh

```diff
@@ -66,6 +66,10 @@ pushd ${TMP_DIR}
   "${AVIFENC}" -s 8 "${INPUT_PNG}" --grid 2x2 -o "${ENCODED_FILE_2x2}"
   "${AVIFDEC}" "${ENCODED_FILE_2x2}" "${DECODED_FILE_2x2}"
 
+  echo "Testing monochrome grid with odd width"
```

Comment:
Optional: add "(403)" after "odd width". Or add a comment to remind ourselves that INPUT_PNG is 403 x 302.